stock_lot_sale_tracking/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    'name': 'This addon allows to retrieve all customer deliveries impacted '
```

Comment:
this should be the summary, right? and then you will also need to add the proper name.